### DISCUSSION TITLE
How to resolve this

### DISCUSSION THREAD

**Discussion by Ojas-Naveen** [webdriverio/codemod#83] [Migration Issue]
when I'm trying to migrate the webdriverIO version from v6-v7 I'm getting the following error. can you please help me out with this?
I ran the command is **`npm i --save-dev @wdio/allure-reporter@7 @wdio/cli@7 @wdio/cucumber-framework@7 @wdio/local-runner@7 @wdio/spec-reporter@7 @wdio/sync@7 webdriverio@7`**

i'm using node version 16

**error An unexpected error occurred: "ENOENT: no such file or directory, symlink '../../bin/gherkin-utils' -> '/Users/user/Library/Caches/Yarn/v4/npm-@cucumber-gherkin-utils-8.0.0-8730d001f16ab9ff6e41febf72575d1c63477edb/node_modules/@cucumber/gherkin-utils/.bin/@cucumber/gherkin-utils'".**

**Comment by christian-bromann** [comment]
> @wdio/sync@7

I don't recommend using sync mode anymore. It is deprecated. Not sure what causes this error, maybe Yarn related?
